### DISCUSSION TITLE
Release on PyPi?

### DISCUSSION THREAD

**Discussion by johnandersen777** [intel/scikit-learn-intelex#498] [General]
Are there any plans to release the daal4py package on PyPi? This would be helpful in situations were a package depends on daal4py and is installed with `pip`: https://github.com/intel/dffml/issues/766

**Comment by PivovarA** [comment]
Hi @pdxjohnny 
yes, we are planning to distribute daal4py through PyPi, but in order to do that we need to solve some difficulties in distribution.
We are working on it.

**Comment by johnandersen777** [comment]
Any update on this?

**Comment by tomashek** [comment]
We hope to have this done before the end of the year.

**Comment by johnandersen777** [comment]
Any update on this?

**Comment by PivovarA** [comment]
Hi @pdxjohnny 
We are working on this right now. We plan to finish with a Pypi release by the end of February

**Comment by PetrovKP** [comment]
Hi @pdxjohnny  

daal4py is already available at the [Python Package Index](https://pypi.org/project/daal4py/)

You can install:
```bash
pip install daal4py
```
At the moment we have the following support matrix:

| OS / Python version     | **Python 36** | **Python 37**  | **Python 38**| **Python 39**|
| :-----------------------| :-----------: | :------------: | :-----------:| :-----------:|
|    **Linux**            |   ✔️         |     ✔️         |     ✔️      |      ✔️      |
|    **Windows**          |   ✔️         |     ✔️         |     ✔️      |      ✔️      |
|    **OsX**              |   ✔️         |     ✔️         |     ✔️      |      ❌      |

**Comment by johnandersen777** [comment]
Awesome! Thank you!!!!
